modules/swagger-codegen/src/main/resources/python/tox.mustache

```diff
@@ -1,10 +1,20 @@
 [tox]
+{{^asyncio}}
 envlist = py27, py3
+{{/asyncio}}
+{{#asyncio}}
+envlist = py3
```

Comment:
`asyncio` with coroutine function definition requires Python 3.5+. I also switched from `nosetest` to `pytest` which supports asyncio.